File: C++/excel-sheet-column-title.cpp
class Solution {
public:
    std::string convertToTitle(int n) {
        std::string title;
        do {
            title += ((n-1)%26)+'A';
            n = (n-1)/26;
        } while(n);
        std::reverse(title.begin(), title.end());
        return title;
    }
};

File: Python/excel-sheet-column-title.py
class Solution:
    # @param {integer} n
    # @return {string}
    def convertToTitle(self, n):
        s = ''

        s += chr((n-1)%26 +65)
        n = (n-1)/26
            
        while n>0:
            s += chr((n-1)%26 +65)
            n = (n-1)/26
        
        return s[::-1]
